amundsen_application/static/js/components/TableDetail/EditableSection/tests/index.spec.tsx

```diff
@@ -48,6 +49,7 @@ describe("EditableSection", () => {
     const customTitle = "custom title";
     const { wrapper, props } = setup({ title: customTitle }, <TagInput/>);
 
+    // TODO how do I test for readOnly?
```

Comment:
heads up, I should still add a test for the readOnly=true case if I can figure it out! Will probably need to reach out to you on slack

Reply:
It looks like this test suite currently didn't test for the button to be rendered. What we'd want to do is create a case where we have `readOnly` as false and verify the edit button exists, any another where we have `readOnly` as true and verify that the edit button doesn't exist.

Reply:
I think i got it up and running with 2 tests now. Did this:
```    it("renders button when readOnly=false", () => {
      expect(wrapper.find(".btn").length).toEqual(1);
    });

    it("does not render button when readOnly=true", () => {
      const { wrapper } = setup({readOnly: true}, <TagInput/>);
      expect(wrapper.find(".btn").length).toEqual(0);
    });```